webapp/client/src/stories/FormFieldIntegerInput.stories.js

```diff
@@ -0,0 +1,99 @@
+import React from "react";
+
+import FormFieldIntegerInput from "../components/FormFieldIntegerInput";
+import StepForm from "../components/StepForm";
+import { Default as StepFormDefault } from "./StepForm.stories";
+
+export default {
+  title: "Form Fields/IntegerInput",
+  component: FormFieldIntegerInput,
+};
+
+function Template(args) {
+  return (
+    <StepForm {...StepFormDefault.args}>
+      <FormFieldIntegerInput {...args} />
+    </StepForm>
+  );
+}
+
+function TemplateLong(args) {
+  const style = {
+    height: 1000,
+  };
+  return (
+    <StepForm {...StepFormDefault.args}>
+      <FormFieldIntegerInput {...args} />
+      <div style={style} />
+    </StepForm>
+  );
+}
+
+export const Default = Template.bind({});
+export const WithValue = Template.bind({});
+export const WithErrors = Template.bind({});
+export const WithDetails = Template.bind({});
+export const WithAllLabelExtras = Template.bind({});
+export const WithMaxWidth = Template.bind({});
+export const WithMaxLength = Template.bind({});
+export const WithMaxLengthAndWidth = Template.bind({});
```

Comment:
Not sure I like this style of defining variants at the top and filling them in later. What's the motivation for it?

Reply:
None, really, it just looks prettier to me. :D